E1a-ARSE/ALIN01/README.md

```diff
@@ -168,4 +179,21 @@ Considering the aim of this test, other **optional** results, not subject to the
 | --------------- | -------------------- | ------- | ------- | ------------ | -------------- |
 | IfcSite         |                      | 1       | 1       | IfcAlignment | Road alignment |
 
+### Project global positioning
+
+<details><summary>Click to expand</summary>
+> **Acceptance criteria**: For the **Spatial containment** capability, the validation procedure must verify that a Spatial Element of the requested type contains (via `IfcRelContainedInSpatialStructure`) exactly a given number of Elements of the requested type, no more and no less.
+
+| Element          | Attribute        | Value     | Comment |
+| ---------------- | ---------------- | --------- | ------- |
+| IfcMapConversion | Eastings         | 24474600  |         |
+| IfcMapConversion | Northings        | 6655000   |         |
+| IfcMapConversion | OrthogonalHeight | 0         |         |
+| IfcMapConversion | XAxisAbscissa    | 1         |         |
+| IfcMapConversion | XAxisOrdinate    | 0         |         |
+| IfcMapConversion | Scale            | 1         |         |
+| IfcProjectedCRS  | Name             | EPSG:3878 |         |
+| IfcProjectedCRS  | GeodeticDatum    | EPSG:3878 |         |
+| IfcProjectedCRS  | VerticalDatum    | EPSG:3900 |         |
```

Comment:
See https://github.com/buildingSMART/IFC4.3.x-development/issues/21

I think it is not OK to have `IfcProjectedCRS::Name` = `IfcProjectedCRS::GeodeticDatum`.

Reply:
Should it be "ETRS89" or "European Terrestrial Reference System 1989" or "https://epsg.io/6258-datum" or doesn't it matter?

Reply:
Can you check now?

Reply:
I'd go for `EPSG:6258`.